src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -136,13 +136,6 @@ public function testSet()
         $this->assertEquals($foo, $sc->get('foo'), '->set() sets a service');
     }
 
-    public function testSetWithNullResetTheService()
```

Comment:
this test should not be removed but be marked as ``@group legacy``, as it covers the legacy feature